gobblin-hive-registration/src/main/java/org/apache/gobblin/hive/HiveMetastoreClientPool.java

```diff
@@ -66,6 +69,10 @@
 
   public static final String POOL_MIN_EVICTABLE_IDLE_TIME_MILLIS = "pool.min.evictable.idle.time.millis";
 
+  public static final String POOL_HIVE_EXTRA_CONFIG_PREFIX = "hive.extraConfig.";
```

Comment:
Can you add javadoc and provide an example of how/when additional configs need to be set?